src/dialogs/more-info/components/lights/dialog-light-color-favorite.ts

```diff
@@ -82,19 +81,19 @@ class DialogLightColorFavorite extends LitElement {
     }
 
     this._modes = modes;
+
+    this._mode =
+      this._color && "color_temp_kelvin" in this._color
+        ? "color_temp"
+        : "color";
```

Comment:
I believe this needs to be checked against `_modes`? E.g. a CCT bulb that is `off` is going to default into "color" mode with this, which it does not support.

Reply:
If color is defined, we check the color to set the mode otherwise, we will take the first available mode.

Reply:
Looks better now, if you're finished I approve (still draft?)

Reply:
Ready to review 👍